api/router.go

```diff
@@ -125,6 +124,9 @@ func (mh *mainHandler) handle(w http.ResponseWriter, r *http.Request) error {
 			// Origin (with port) matches Host.
 		case originURL.Hostname() == r.Host:
 			// Origin (without port) matches Host.
+		case originURL.Scheme == "chrome-extension":
+			// Allow access for the browser extension
+			// TODO(ppacher): can we improve that check here?
```

Comment:
This will allow any extension to communicate with the Portmaster.
I understand that extensions are things that a user installs, but I fear that many users still do this without too much evaluation.
Is there anything else that the origin contains in this case that we can use?
Also, do we need to add something for Firefox here?